modules/cluster-logging-updating-logging-to-current.adoc

```diff
@@ -0,0 +1,218 @@
+:_content-type: PROCEDURE
+[id="cluster-logging-updating-logging-to_current_{context}"]
+= Updating OpenShift Logging to the current version
+
+To update OpenShift Logging to the current version, you change the subscriptions for the OpenShift Elasticsearch Operator and Red Hat OpenShift Logging Operator.
+
+[IMPORTANT]
+====
+You must update the OpenShift Elasticsearch Operator _before_ you update the Red Hat OpenShift Logging Operator. You must also update _both_ Operators to the same version.
+====
+
+
+If you update the operators in the wrong order, Kibana does not update and the Kibana custom resource (CR) is not created. To work around this problem, you delete the Red Hat OpenShift Logging Operator pod. When the Red Hat OpenShift Logging Operator pod redeploys, it creates the Kibana CR and Kibana becomes available again.
+
+.Prerequisites
+
+* The {product-title} version is 4.7 or later.
+
+* The OpenShift Logging status is healthy:
++
+** All pods are `ready`.
+** The Elasticsearch cluster is healthy.
+
+* Your link:https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-restore.html[Elasticsearch and Kibana data is backed up.]
+
+.Procedure
```

Comment:
Some of the lettered sub-steps are missing letters.

https://50781--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-upgrading.html